frigate/ffmpeg_presets.py

```diff
@@ -91,7 +91,7 @@ def get_selected_gpu(self) -> str:
 PRESETS_HW_ACCEL_SCALE = {
     "preset-rpi-64-h264": "-r {0} -vf fps={0},scale={1}:{2}",
     "preset-rpi-64-h265": "-r {0} -vf fps={0},scale={1}:{2}",
-    FFMPEG_HWACCEL_VAAPI: "-r {0} -vf fps={0},scale_vaapi=w={1}:h={2}:format=nv12,hwdownload,format=nv12,format=yuv420p",
+    FFMPEG_HWACCEL_VAAPI: "-r {0} -vf fps={0},scale_vaapi=w={1}:h={2},hwdownload,eq=gamma=1.05",
```

Comment:
Could we make the gamma conditional based on the ffmpeg version?

Reply:
I don't think we want the presets here to be aware of the users config and in the case that ffmpeg is path linked it becomes more complicated to handle all cases. I would rather make this the default for now, we know that ffmpeg 7.1 seems to have changed the way that the yuvj pixel format is handled so we can likely take this out and handle it explicitly once we update to 7.1

Reply:
Agreed, 7.1 is where we want to land eventually. Perhaps just an undocumented way to prevent the gamma from being applied is all I'm looking for.

Reply:
Bind mounting the file without the changes for my setup should be fine for now.